<launch>
  <arg name="gui" default="true"/>
  <arg name="state_rate" default="10"/>
  <param name="use_gui" value="$(arg gui)"/>
  <param name="rate" value="$(arg state_rate)"/>
  <param name="robot_description"
    command="$(find xacro)/xacro --inorder '$(find crane_x7_description)/urdf/crane_x7.urdf.xacro'"
    />
  <rosparam param="source_list">["joint_states_source"]</rosparam>
  <node name="vis_joint_publisher" pkg="crane_x7_examples" type="joint_publisher1.py"/>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find crane_x7_description)/config/urdf.rviz"/>
</launch>

